src/index.js

```diff
@@ -198,20 +197,11 @@ class HMIApp extends React.Component {
         }
 
         var resetPeriodSelector = undefined;
-        if (this.props.activeAppState && this.props.activeAppState.videoStreamingCapability.length) {
             resetPeriodSelector = (<div className="reset-period-selector">
                 <label>Reset period, ms:</label><br/>
                 <input type="text" value={this.props.resetPeriodValue} onChange={this.changeResetPeriod} />
             </div>);
             let state = store.getState();
```

Comment:
@LiliiaShlikhtLuxoft Looks like `state` is unused

Reply:
@ValeriiMalkov fixed.